app/assets/stylesheets/components/user_header.scss

```diff
@@ -0,0 +1,19 @@
+/* Header Styles */
+
+.user-header > a {
```

Comment:
@eric-ls I'm not sure why this makes the entire background-color `$white` (in the pic I attached).... but I was thinking this would change all `.user-header`'s `a` tagged elements as `$white`